docs/cicd-study.md

```
@@ -0,0 +1,246 @@
+배포 묶음 파일(`deploy.zip`)을 임시로 저장하는 용도로 사용합니다.
+CodeDeploy는 S3에서 zip을 받아와야만 EC2에 배포할 수 있습니다.
+
+```
+GitHub Actions
+  └─ deploy.zip 생성 (appspec.yml + scripts/)
+       └─ S3 업로드
+            └─ CodeDeploy가 S3에서 받아서 EC2에 전달
+```
+
+**deploy.zip 안에 들어있는 것:**
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)